### PR TITLE
Refactoring/mvp refactoring

### PR DESCRIPTION
**Pull request by michaelputnam67**
# Description

Finalize refactoring of our index.html, classes, and scripts files. Add semicolons and group functions, query selectors, and event handlers together in a logical order.

Addresses #6 (issue)

## Type of change

- [x] Refactoring

# How Has This Been Tested?

- [x] DevTools

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -44,22 +44,6 @@ <h4>Meal Type</h4>
       <section class="recipes-wrapper">
         <input type="search" placeholder="Search by name.." id="searchBar">
         <section class="recipes-section" id="recipesSection">
-          <section class="recipe-preview">
```
Comment (angelewilliams) [inline dist/index.html:47]: Good call removing this.

File: src/classes/RecipeRepository.js [hunk 2]
```diff
@@ -1,39 +1,38 @@
...
   addRecipes(recipeData, ingredientData) {
     return recipeData.map((data) => {
-      return new Recipe(data, ingredientData)
-    })
-  }
+      return new Recipe(data, ingredientData);
+    });
+  };
 
   getTags() {
```
Comment (angelewilliams) [inline src/classes/RecipeRepository.js]: Think we need to add recipeData in as parameter there. @michaelputnam67 can you please add and re-push?

File: src/scripts.js [hunk 3]
```diff
@@ -8,230 +8,201 @@ import './images/icon_fire_symbol_lit.png';
...
     } else {
-      createRecipePreview(recipeRepository.allRecipes)
-    }
-  }
+      createRecipePreview(recipeRepository.allRecipes);
+    };
+  };
 
-  //🌎 GLOBAL VARIABLE REQUIRED
   let identifyRecipe = (e, recipeRepository, user) => {
```
Comment (angelewilliams) [inline src/scripts.js:151]: Looking much better!

File: src/scripts.js [hunk 4]
```diff
@@ -254,97 +225,102 @@ var createRecipePreview = (recipes, e) => {
...
+    return;
+  };
   const tempRecipesArray = recipeRepository.filterByName(inputName);
-  createRecipePreview(tempRecipesArray)
-}
-//😬
+  createRecipePreview(tempRecipesArray);
+};
+
 const displayPopUp = (recipe) => {
```
Comment (angelewilliams) [inline src/scripts.js:267]: Nice job splitting this up

File: src/classes/RecipeRepository.js [hunk 5]
```diff
@@ -1,39 +1,38 @@
...
-      return new Recipe(data, ingredientData)
-    })
-  }
+      return new Recipe(data, ingredientData);
+    });
+  };
 
-  getTags() {
-    this.tags = recipeData.reduce((acc, recipe) => {
+  getTags(recipeData) {
```
Comment (angelewilliams) [inline src/classes/RecipeRepository.js:15]: Thanks for the quick fix!

**Review by angelewilliams — APPROVED** [review]

File: dist/index.html (hunk 1 above)
Comment (michaelputnam67) [reply dist/index.html:47]: Thank you for the positive feedback!

File: src/classes/RecipeRepository.js (hunk 2 above)
Comment (michaelputnam67) [reply src/classes/RecipeRepository.js]: Fixed your request. Thank you for pointing this out!

**Pull request merged**